MultimodalQnA/docker_compose/intel/hpu/gaudi/README.md

```diff
@@ -37,8 +37,6 @@ export WHISPER_PORT=7066
 export WHISPER_SERVER_ENDPOINT="http://${host_ip}:${WHISPER_PORT}/v1/asr"
 export MAX_IMAGES=1
 export WHISPER_MODEL="base"
-export ASR_ENDPOINT=http://$host_ip:$WHISPER_PORT
-export ASR_PORT=9099
 export ASR_SERVICE_PORT=3001
 export ASR_SERVICE_ENDPOINT="http://${host_ip}:${ASR_SERVICE_PORT}/v1/audio/transcriptions"
 export DATAPREP_MMR_PORT=6007
```

Comment:
On line 122 and 221, could you changes those headers from ASR to Whisper Server or something?